package.json

```diff
@@ -28,10 +28,10 @@
   "scripts": {
     "browserstack": "node e2e/browserstack.js -c e2e/browserstack.conf.js -e browserstack --skiptags wip",
     "build": "rm -rf build && run-s \"getcapabilities -- {1}\" \"build:config -- {1}\" --",
-    "postbuild": "run-p build:css build:js build:tests && grunt build site",
+    "postbuild": "run-p build:css build:js build:tests && npx grunt build site",
     "build:css": "node ./tasks/buildCSS",
     "build:config": "bash -c 'tasks/buildOptions.sh ${1}' 0",
-    "postbuild:config": "grunt config",
+    "postbuild:config": "npx grunt config",
```

Comment:
Do we need to `npm i --save-dev npx` (Add it to the package devDependencies)? and/or do we need to add npx install globally in the travis.yml file?

Reply:
It comes packaged with node.js but this is only something I've discovered recently.